extensions/amp-image-slider/0.1/amp-image-slider.css

```diff
@@ -122,10 +123,22 @@ amp-image-slider amp-img > img {
   transition: all 0.3s;
 }
 
-.i-amphtml-image-slider-hint-icon {
-  font-weight: bold;
-  font-size: 3em;
-  color: white;
-  text-shadow: black 2px 0 10px;
-  text-align: center;
+.i-amphtml-image-slider-hint-left-arrow {
```

Comment:
When you expose an external css class for developers, you will want to !important all the attributes here that you do not wish people to override. =)

Reply:
Will do for P1